src/components/Drawer/Drawer.styles.js

```diff
@@ -14,14 +14,13 @@ export const DrawerContainer = styled.div`
   bottom: 0;
   right: 0;
   visibility: hidden;
-  transform: translateY(-100%);
   opacity: 0;
   transition: opacity 300ms ${constants.easing.easeInOutQuad};
   overflow: hidden;
   z-index: -1;
 
   &.drawer__content--open {
-    transform: translateY(0);
+    display: block;
```

Comment:
What do you need display: block for? The element has position: fixed, setting a display is not required. (I did not find a display: none rule neither)